extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -151,17 +151,14 @@ export class DoubleclickA4aEligibility {
     }
     let experimentId;
     const urlExperimentId = extractUrlExperimentId(win, element) || '';
-    if (hasUSDRD &&
-        ![DOUBLECLICK_EXPERIMENT_FEATURE.USDRUD_EXPERIMENT,
-          DOUBLECLICK_EXPERIMENT_FEATURE.USDRUD_CONTROL].includes(
-            URL_EXPERIMENT_MAPPING[urlExperimentId])) {
-      return false;
-    }
-    if (useRemoteHtml && !element.getAttribute('rtc-config') &&
-        ![DOUBLECLICK_EXPERIMENT_FEATURE.REMOTE_HTML_EXPERIMENT,
-          DOUBLECLICK_EXPERIMENT_FEATURE.REMOTE_HTML_CONTROL].includes(
-            URL_EXPERIMENT_MAPPING[urlExperimentId])) {
-      return false;
+    if (![DOUBLECLICK_EXPERIMENT_FEATURE.USDRUD_EXPERIMENT,
```

Comment:
Should we create an experiment branch for both so we can measure the summation effect of removing delayed fetch?